src/js/views/admin/report/report_views.js

```diff
@@ -0,0 +1,54 @@
+import Radio from 'backbone.radio';
+import hbs from 'handlebars-inline-precompile';
+
+import { View } from 'marionette';
+
+import PreloadRegion from 'js/regions/preload_region';
+
+import './report.scss';
+
+// const ReportView = View.extend({});
```

Comment:
what is `ReportView`?

Reply:
I figured it might be necessary to display the report. But that will probably behave the way forms do, where we just build the url for the report data and insert it into the iframe, right? I can remove that.